projects/storefrontlib/layout/a11y/keyboard-focus/lock/lock-focus.directive.ts

```diff
@@ -129,10 +129,11 @@ export class LockFocusDirective
        * we persist the group key to the children, so that they can taken this
        * into account when they persist their focus state.
        */
-      if (!!this.group) {
+      if (this.group) {
```

Comment:
We can keep `!!this.group`. It just helps convert it to a boolean in the end whether it is an empty string, null, or undefined, it won't ever go inside